dftimewolf/lib/state.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, config):
     self.global_errors = []
     self.input = []
     self.output = []
-    self.store = {}
+    self.store = []
```

Comment:
I'm a little worried about the "just put everything in a list and iterate over it". I guess this is OK for small numbers of things in the store, but what about still using a dict, keyed by CONTAINER_TYPE? It looks like be a lost faster, with not much extra complexity.